data/udev/rules.d/66-snapd-autoimport.rules

```diff
@@ -1,3 +1,3 @@
 # probe for assertions, must run before udisks2
-ACTION=="add", SUBSYSTEM=="block" \
+ACTION=="add", SUBSYSTEM=="block", KERNEL!="loop*", KERNEL!="ram*" \
```

Comment:
We already skip /dev/loop & /dev/ram mount points:
https://github.com/snapcore/snapd/blob/master/cmd/snap/cmd_auto_import.go#L83
And if `--mount=/dev/loop*` was passed explicitly, we skip that too:
https://github.com/snapcore/snapd/blob/master/cmd/snap/cmd_auto_import.go#L267

Leaves me with a gut feeling that the `auto-import` code does more than needed, or does the same thing multiple times, and filtering out the devices in udev only masks the problem.

Reply:
That is interesting. But, I think it is preferable to avoid running the process at all if unneeded. It actually takes some time and it is hitting us in a low performance customer device (with probably too many /dev/loop* devices though).